core/src/main/java/io/github/fourlastor/game/demo/turns/PickMove.java

```diff
@@ -56,6 +70,7 @@ public interface Factory {
         PickMove create(Unit unit);
     }
 
+    // sheerst - could these types of classes somehow be removed?
```

Comment:
they're really convenient to be able to remove them in `exit` as we can check for type